java/src/jmri/jmrit/roster/swing/RosterFrameAction.java

```diff
@@ -50,7 +50,7 @@ public RosterFrameAction(String pName, boolean allowQuit) {
         this.allowQuit = allowQuit;
     }
 
-    boolean allowQuit = true;
+    boolean allowQuit = false;
```

Comment:
What does this change do?

Reply:
Normally, when you close the main DecoderPro window, it prompts you that this will exit the application; if you have several main DP windows open and close just one, that prompt-and-quit behavior doesn't happen.  The `allowQuit` argument in some ctors for RosterFrameAction controls this.

(`allowQuit` is poorly named.  It should probably be something like `showQuitDialogOnClose`)

Unfortunately, when you opened a DP window from PanelPro (and now LccPro), you get the same behavior when you try to close that DP window.  (There are several paths to opening a DP window, and at this point I don't remember which of them had this problem) Fixing that involved changing the default for `allowQuit` for ctors that don't carry the `allowQuit` argument, hence this change, along with changes to the way that DP opens its initial window.